eZ/Publish/Core/settings/repository.yml

```diff
@@ -63,88 +76,220 @@ services:
         lazy: true
 
     ezpublish.api.service.content:
-        class: "%ezpublish.api.service.content.class%"
+        alias: ezpublish.signalslot.service.content
+
+    ezpublish.api.service.content_type:
+        alias: ezpublish.signalslot.service.content_type
+
+    ezpublish.api.service.field_type:
+        alias: ezpublish.signalslot.service.field_type
+
+    ezpublish.api.service.role:
+        alias: ezpublish.signalslot.service.role
+
+    ezpublish.api.service.object_state:
+        alias: ezpublish.signalslot.service.object_state
+
+    ezpublish.api.service.url_wildcard:
+        alias: ezpublish.signalslot.service.url_wildcard
+
+    ezpublish.api.service.url_alias:
+        alias: ezpublish.signalslot.service.url_alias
+
+    ezpublish.api.service.user:
+        alias: ezpublish.signalslot.service.user
+
+    ezpublish.api.service.search:
+        alias: ezpublish.signalslot.service.search
+
+    ezpublish.api.service.section:
+        alias: ezpublish.signalslot.service.section
+
+    ezpublish.api.service.trash:
+        alias: ezpublish.signalslot.service.trash
+
+    ezpublish.api.service.location:
+        alias: ezpublish.signalslot.service.location
+
+    ezpublish.api.service.language:
+        alias: ezpublish.signalslot.service.language
+
+    # Inner API
+    ezpublish.api.service.inner_content:
+        class: %ezpublish.api.service.content.class%
```

Comment:
I recommend surrounding `%parameters%` with double quotes to avoid YAML deprecation warnings when we switch to Symfony `3.1` in the future.